src/Helpers/AliasStatus.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Visiosoft\SiteModule\Helpers;
+
+use Visiosoft\SiteModule\SiteModuleInterface;
+
+abstract class AliasStatus
+{
+    const WAITING = 0;
```

Comment:
good abstract class @emrullahardc 

@spektra2147

Reply:
thank you.